### ISSUE TITLE
Lazy Loading is not properly working

### ISSUE THREAD

**Issue by igmoweb** [humanmade/altis-media#62]
Lazy loading module is not blurring the images in Concentra (here's a sample: https://concentra-staging.aws.hmn.md/about/news/lazy-loading-test/). Apparently, Gaussholder data attributes are missing. Quoting @roborourke :

> we now support saving tachyon URLs directly in the content - gaussholder was relying on the generated image thumbnail names but we don’t actually generate intermediate sizes any more

**Comment by roborourke** [comment]
@igmoweb so digging a bit further it seems the problem only affects full size images, I just tested with a large image in your example post above.

I think it's something we may need to fix in Tachyon itself but I confirmed a solution using the following filter:

```php
add_filter( 'tachyon_image_downsize_string', function ( $tachyon_args, $args ) {
	if ( $args['size'] === 'full' && isset( $tachyon_args[ $args['transform'] ] ) ) {
		unset( $tachyon_args[ $args['transform'] ] );
	}
	return $tachyon_args;
}, 10, 2 );
```

**Comment by roborourke** [comment]
Little update, got 2 PRs to help with this, just waiting for review currently:

- https://github.com/humanmade/tachyon-plugin/pull/54
- https://github.com/humanmade/smart-media/pull/39

**Comment by roborourke** [comment]
This one got fixed in version 3.0.3 of the media module 👍
